JavaScriptTasks/Session 5 - OOPInheritance(5-3-2020)/SantaClausableInterface.js

```diff
@@ -0,0 +1,50 @@
+// PROBLEM STATEMENT :SantaClausable Interface
+
+// http://www.codewars.com/kata/santaclausable-interface
+// SOLUTION:
+
+ 
+function isSantaClausable(obj) {
+
+  const condition1 =  typeof obj['sayHoHoHo'] == 'function';
+   const condition2 =  typeof obj['distributeGifts'] == 'function';
+    const condition3 = typeof obj['goDownTheChimney'] == 'function';
+
+  
+  if((condition1 === true && condition2 === true && condition3 ===true))
```

Comment:
please remove this if else block, directly return condition1 === true && condition2 === true && condition3 ===true

Reply:
made the suggested changes